example/src/components/RoomParticipant.tsx

```diff
@@ -18,23 +18,32 @@ import { PinButton } from './buttons/PinButton';
 
 type RoomParticipantProps = {
   participant: Membrane.Participant;
+  trackId?: string;
   onPinButtonPressed?: (string) => void;
   focused?: boolean;
   pinButtonHiddden?: boolean;
   tileSmall?: boolean;
 };
 
 export const RoomParticipant = ({
-  participant: { metadata, tracks, type, id },
+  participant,
+  trackId = '',
   onPinButtonPressed = (string) => {},
   focused = false,
   pinButtonHiddden = false,
   tileSmall = false,
 }: RoomParticipantProps) => {
+  const { metadata, tracks, type } = participant;
+
   const [showPinButton, setShowPinButton] = useState(false);
   const isPinButtonShown = useRef(false);
-  const videoTrack = tracks.find((t) => t.type === 'Video');
-  const audioTrack = tracks.find((t) => t.type === 'Audio');
+
+  const videoTrack = tracks.find((t) => t.id === trackId);
+  const videoTrackType = videoTrack?.metadata.type;
+  const audioTrack =
+    !videoTrack || videoTrackType === 'camera'
```

Comment:
It's not clear what are you doing here: if there is no video track or the videotrack is from camera then search for audio track? Why? You already have a check if the track is screenshared

Reply:
I never realised I could just that screenshared check from before, good point. Should be simpler now.